### DISCUSSION TITLE
Calculated Values - Add support for Date Fields

### DISCUSSION THREAD

**Discussion by kozziag** [Innofactor/XrmToolBox.BulkDataUpdater#153] [Ideas]
Can there be a support for Date Fields. At this time only text/string fields are supported for calculated value updates. It would be nice to also add support for date fields. If modifiedon date/time field can also be considered it will help.
Thanks in advance.

**Comment by rappen** [comment]
Thanks @kozziag 

I will take a look at the date/time columns targets, and see how we can modify them.
I'm glad to get some ideas of what you want to do?
